modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -161,14 +165,16 @@ You can repeat these steps to add multiple key values.
 [NOTE]
 Custom Metrics Connector requires all *Facts* to be sent from every operation, even if they contain empty or 0 value in them.
 
-== Configure Custom Metrics Values Using Anypoint Studio
+== Configure Custom Metrics from Anypoint Studio
```

Comment:
candidate for documentation in Studio docs vs. here, but saving that for later. will talk to CrisP about it.